### PR TITLE
docs: improve Top Level Docs

### PR DESCRIPTION
**Pull request by klaarseSICKAG**
# Pull Request

## 📖 Description

This PR tries to optimize the top level documentation in Storybook.

### 🎫 Issues

#304 

## 👩‍💻 Reviewer Notes

Start the docs server:  cd packages/docs && pnpm start
or goto: https://64f819d70a69cb8728e06daf-wnunbsasih.chromatic.com/?path=/docs/welcome--docs

## 📑 Test Plan

Go through the top level docs:
 - Welcome (biggest changes)
 - Goals and guidelines (only title changed)
 - Detailed overview (title and description changed)

## ✅ DoD

- [x] I checked that the documentation is better to read than before
- [x] I checked for spelling errors
- [x] I checked the segmentation of files and chapters
- [x] I checked that all links are working
- [x] I checked that the layout is ok

### CODE REVIEW

File: packages/_private/e2e-demo-test/playwright-report/index.html
Comment (schilchSICKAG) [inline packages/_private/e2e-demo-test/playwright-report/index.html:1]: I don´t think this belongs here. Should we add `playwright-report` to `.gitignore`?

File: packages/docs/stories/Welcome.mdx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+# Synergy Design System ![](/heart.svg)
+
+Welcome to the Synergy Design System: the centralized, living style guide for [SICK](https://www.sick.com) application user interfaces.
+
+Synergy gives you a head start for your application. Use colors, styles, components and assets to quickly spin-up your application in development ✍ and design 🎨 at the same time.
+Build multiple apps with different frameworks sharing the same UI components.
+
+## Sources &lt;/&gt;
+<div style={{display:"flex", alignItems:"center"}}><img src="/github.png" height="30"></img> &nbsp;The source code is available on&nbsp;[GitHub](https://github.com/synergy-design-system/synergy-design-system) </div><br/>
```
Comment (schilchSICKAG) [inline packages/docs/stories/Welcome.mdx:15]: Just a minor nitpick, but could we also add the links to the `` tags themselfs?

File: packages/docs/stories/Welcome.mdx [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+<div style={{display:"flex", alignItems:"center"}}><img src="/figma.png" height="30"></img> &nbsp;The Figma files are available on&nbsp;[Figma](https://www.figma.com/file/bZFqk9urD3NlghGUKrkKCR/Synergy-Digital-Design-System) &nbsp; for SICK users only.</div>
+
+
+## Getting started 🛫
+<div style={{display:"flex", alignItems:"center", margin:"8px"}}><img src="/html5.png" height="35"></img>&nbsp; with vanilla JS:&nbsp;[click here](../?path=/docs/packages-components-web-components--docs) <br/></div>
+<div style={{display:"flex", alignItems:"center"}}><img src="/ng.webp" height="50"></img> with Angular:&nbsp;[click here](../?path=/docs/packages-components-angular--docs) <br/></div>
```
Comment (schilchSICKAG) [inline packages/docs/stories/Welcome.mdx:21]: This link does not work

File: packages/docs/stories/Welcome.mdx [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+<div style={{display:"flex", alignItems:"center"}}><img src="/react.webp" height="50"></img> with React:&nbsp;[click here](../?path=/docs/packages-components-react--docs) <br/></div>
```
Comment (schilchSICKAG) [inline packages/docs/stories/Welcome.mdx:22]: This link does not work

File: packages/docs/stories/Welcome.mdx [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+<div style={{display:"flex", alignItems:"center"}}><img src="/vue.webp" height="50"></img> with VUue:&nbsp;[click here](../?path=/docs/packages-components-vue--docs) <br/></div>
```
Comment (schilchSICKAG) [inline packages/docs/stories/Welcome.mdx:23]: And this does not work, too :(

**Review by schilchSICKAG — CHANGES_REQUESTED** [review]
Other than the failing links, I think it is a very good addition to the previous version of the start page. When it comes to page design, I think @eileenwegner has already added some comments.

**Comment by reichst1SICKAG** [comment]
Link color shuld be var(--syn-color-primary-600), which is the SICK blue. It is primary-500.

File: packages/docs/stories/Welcome.mdx [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+
+<img src="/synergy-s.png" height="100" style={{float: 'right', marginTop: 30, marginBottom: 30}}></img>
+
+# Welcome to the Synergy Design System
+<div style={{fontSize:"x-large"}} class="sb-unstyled">
+The centralized, living style guide for **[SICK](https://www.sick.com)** application user interfaces.
+</div>
+<div style={{fontSize: "medium"}} class="sb-unstyled">
+Synergy gives you a head start for your application. Use colors, styles, components and assets to quickly spin-up your application in development ✍ and design 🎨 at the same time.
+Build multiple apps with different frameworks sharing the same UI components.
```
Comment (reichst1SICKAG) [inline packages/docs/stories/Welcome.mdx:15]: Change oder adapt to the content on mosaic `About Synergy`?

The Synergy Design System empowers teams to create brand-compliant, adaptable and accessible user-interfaces across all digital products and solutions. The design system combines design with development, generating a cohesive and aligned experience for our users.

File: packages/docs/stories/Welcome.mdx [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+</div>
+
+<br/><br/>
+
+<Chapter title="Sources">
+  <SymbolLine title={<img src='/github.png' height='30'></img>}>The source code is available on&nbsp;[GitHub](https://github.com/synergy-design-system/synergy-design-system)</SymbolLine><br/>
+  <SymbolLine title={<img src="/figma.png" height='30'></img>}>The Figma files are available on&nbsp;[Figma](https://www.figma.com/file/bZFqk9urD3NlghGUKrkKCR/Synergy-Digital-Design-System) &nbsp; for SICK users only.</SymbolLine>
+</Chapter><br/>
```
Comment (reichst1SICKAG) [inline packages/docs/stories/Welcome.mdx:23]: double blank space after the Figma link

File: packages/docs/stories/Welcome.mdx [hunk 7]
```diff
@@ -0,0 +1,83 @@
...
+- Light and dark themes are included
+- Material icons
+- Support for all major browsers; tested with every release.
+- Basic CSS styles for an easy application start
+- Templates for typical and recurring UIs which can be copied.
+</Chapter><br/>
+
+<Chapter title="Foundations">
+- The [web-components](https://developer.mozilla.org/en-US/docs/Web/API/Web_components) are build upon popular technologies, especially [Lit](https://lit.dev) and [Shoelace](https://shoelace.style) 🙏
+- [Material Icons](https://fonts.google.com/icons) from Google 🙏
```
Comment (reichst1SICKAG) [inline packages/docs/stories/Welcome.mdx:49]: the icons look like praying hands to me, can you change/remove them?

File: packages/docs/stories/Welcome.mdx [hunk 8]
```diff
@@ -0,0 +1,83 @@
...
+</Chapter><br/>
+
+<Chapter title="Trust">
+_Using a base library for an application is and should be a careful and very well-considered process. Besides the technology stack and features another key factor is trust.
+Why do we think you should trust Synergy?_
+- We also considered our base technologies very carefully and decided for the ones which are most popular and easy to maintain if necessary
```
Comment (reichst1SICKAG) [inline packages/docs/stories/Welcome.mdx:55]: remove fill word 'also'

File: packages/docs/stories/Welcome.mdx [hunk 9]
```diff
@@ -0,0 +1,83 @@
...
+- We are building on web standards
+- Every used technology is independent from other technologies as much as possible and meaningful
+- In development, dependencies are updated on a regular basis
+- It is our aim to provide a long lasting library with a stable API.
+- Synergy has a dedicated team with almost a single responsibility into this project
+- All team members are very experienced designers and web developers with a long history in developing design systems
+- Synergy is backed by big and strategically important products at SICK
```
Comment (reichst1SICKAG) [inline packages/docs/stories/Welcome.mdx:62]: Move to top of the list

File: packages/docs/stories/Welcome.mdx [hunk 10]
```diff
@@ -0,0 +1,83 @@
...
+- Synergy is seen by top management as a very important corner stone for a successful and versatile product palette
```
Comment (reichst1SICKAG) [inline packages/docs/stories/Welcome.mdx:63]: Move to top of the list

File: packages/docs/stories/Welcome.mdx [hunk 11]
```diff
@@ -0,0 +1,83 @@
...
+<Chapter title="Trust">
+_Using a base library for an application is and should be a careful and very well-considered process. Besides the technology stack and features another key factor is trust.
+Why do we think you should trust Synergy?_
+- We also considered our base technologies very carefully and decided for the ones which are most popular and easy to maintain if necessary
+- We are building on web standards
+- Every used technology is independent from other technologies as much as possible and meaningful
+- In development, dependencies are updated on a regular basis
+- It is our aim to provide a long lasting library with a stable API.
+- Synergy has a dedicated team with almost a single responsibility into this project
+- All team members are very experienced designers and web developers with a long history in developing design systems
```
Comment (reichst1SICKAG) [inline packages/docs/stories/Welcome.mdx:61]: Move to top of the list

File: packages/docs/stories/Welcome.mdx [hunk 12]
```diff
@@ -0,0 +1,83 @@
...
+- Synergy is backed by big and strategically important products at SICK
+- Synergy is seen by top management as a very important corner stone for a successful and versatile product palette
+- We are putting a lot of effort into quality assurance
+- We are listening very carefully to our stakeholders needs
```
Comment (reichst1SICKAG) [inline packages/docs/stories/Welcome.mdx:65]: Move to top of the list

File: packages/docs/stories/Welcome.mdx [hunk 13]
```diff
@@ -0,0 +1,83 @@
...
+- We made everything transparent: code (open source), roadmap, issues, aims, guidelines, decision, ...
+</Chapter><br/>
+
+<Chapter title="Roadmap">
+What you're seeing here and now is just the status quo of provided deliveries. New features and components are developed continuously.
```
Comment (reichst1SICKAG) [inline packages/docs/stories/Welcome.mdx:70]: I would write: 
Our product roadmap is an up-to-date report on the work we’re doing and planning to do. New features and components are added continuously.

File: packages/docs/stories/GoalsAndGuidelines.mdx [hunk 14]
```diff
@@ -20,9 +20,9 @@ We stay ahead, making our design system more flexible for our brand's future nee
 These principles help us to achieve our goals.
```
Comment (reichst1SICKAG) [inline packages/docs/stories/GoalsAndGuidelines.mdx:20]: Can you use the order like in the presentation: Progressive, Unified, Connected
Or use the file from #internal https://github.com/synergy-design-system/Internal/blob/main/docs/guiding_principles.md

**Comment by burktiSICKAG** [comment]
Remove the "Trust" section. This is not meant for the public.

File: packages/docs/stories/GoalsAndGuidelines.mdx [hunk 15]
```diff
@@ -20,9 +20,9 @@ We stay ahead, making our design system more flexible for our brand's future nee
...
 
 
-## Vision and goals
+## Mission and goals
 
-### Vision
+### Mission
```
Comment (reichst1SICKAG) [inline packages/docs/stories/GoalsAndGuidelines.mdx:25]: The headline of `Goals` below is to big. Should be like `Mission`

File: packages/docs/stories/Welcome.mdx (hunk 1 above)
Comment (klaarseSICKAG) [reply packages/docs/stories/Welcome.mdx]: How would you change the SymbolLine component to achieve that?

File: packages/_private/e2e-demo-test/playwright-report/index.html
Comment (klaarseSICKAG) [reply packages/_private/e2e-demo-test/playwright-report/index.html:1]: this was a relic. Removed it. thx

File: packages/docs/stories/Welcome.mdx (hunk 5 above)
Comment (klaarseSICKAG) [reply packages/docs/stories/Welcome.mdx:13]: you're right. we probably should stick to the same wording. which do you like better?

Comment (klaarseSICKAG) [reply packages/docs/stories/Welcome.mdx:13]: giving it another thought. the page on mosaic is more related to SICK (brand-compliant, digital products, etc.) The description here is less SICK related and fits imho more to the open source / github character. I'll merge it as it is. We can have another chat about it at a later time...

**Review by kirchsuSICKAG — APPROVED** [review]

**Pull request merged**